README.md

```diff
@@ -118,17 +118,21 @@ type  │                    data
      format
 ```
 
-### Diffie-Hellman formats
+### 3. key formats
+
+keys used for encryption
 
 | Type code | Format code | Data length | Format name | Specification |
-|-----------|-------------|-------------|-------------|---------------|
-| 3         | 0           | 32 bytes    | curve25519  |               |
+|:---------:|:-----------:|-------------|-------------|---------------|
+| 3         | 0           | 32 bytes    | envelope-dh |               |
```

Comment:
Would be nice, maybe by linking or mentioning in Specification that this is a curve25519 key

Reply:
DONE
have also renamed this to box2-dm-dh  short for (private group box2, direct message, diffie-hellman)